server/server.go

```diff
@@ -1399,10 +1401,13 @@ func (s *Server) Start() {
 							return
 						case <-t.C:
 							if _, err := ar.Fetch(s.opts.SystemAccount); err != nil {
-							} else if _, err := s.fetchAccount(s.opts.SystemAccount); err == nil {
-								s.Noticef("System account fetched and updated")
-								return
+								continue
 							}
+							if _, err := s.fetchAccount(s.opts.SystemAccount); err != nil {
+								continue
```

Comment:
I meant for the `Fetch()` above. That one was fine to say that if no error, then print and then be done. The implicit else is equivalent of a continue.

Reply:
I know. after the change it seemed cleaner to perform the same error check